torch/csrc/CudaIPCTypes.cpp

```diff
@@ -162,23 +162,23 @@ CudaIPCSentData::CudaIPCSentData(
     // because the main thread may have queued extra work on the stream, which
     // this event will consequently wait for (uselessly).
     cuda_ipc_global_entities.sync_events_used_ ++;
-    C10_CUDA_CHECK(cudaEventCreateWithFlags(
+    C10_HIP_CHECK(hipEventCreateWithFlags(
```

Comment:
@zstreet87 Are these hipified changes being committed to the branch? If so, why? cc @jeffdaily

Reply:
goodness bad habit!  Only the few files I changed for resolving merge conflicts should have been pushed...